### ISSUE TITLE
Deprecate panel

### ISSUE THREAD

**Issue by tfrijsewijk** [dso-toolkit/dso-toolkit#1464]
Dit component wordt nog gebruikt in `\alexia\apps\src\app\compose\common\wizard-steps\DocumentStep.jsx`.

/cc: @MichielDHVD

**Comment by MichielDHVD** [comment]
Hmmm dat zou dus terug moeten komen in een PR12 applicatie. @bramhoosemans  even met het team door laten nemen. En kijken hoe of wat aangepast moet worden.

**Comment by bramhoosemans** [comment]
@tfrijsewijk @MichielDHVD Deze kan nog niet worden ingepland/verwijderd.

**Comment by MichielDHVD** [comment]
Zet ik deze terug naar de backlog op jira en hier.

**Comment by gofix** [comment]
@bramhoosemans @MichielDHVD kunnen we hier inmiddels al wel een keer vanaf?

**Comment by bramhoosemans** [comment]
> @bramhoosemans @MichielDHVD kunnen we hier inmiddels al wel een keer vanaf?

Nee. In het kort: we hebben eerste het Web Component voor 'Accordion' nodig. PR12 maakt gebruik van een panel implementatie voor accordions ivm bootstrap 3.3.7 (legacy). Deze moeten dan eerst worden omgezet.

**Comment by tfrijsewijk** [comment]
Tussentijdse update: Het team van PR12 is bezig met het implementeren van Web Components. Mogelijk dat deze dus binnenkort opgepakt kan worden 🎉

**Comment by tfrijsewijk** [comment]
@DjangoBlok Jij wilt het Panel component herintroduceren. Dit issue komt dan te vervallen

**Comment by tfrijsewijk** [comment]
Zojuist is mij bevestigd dat de laatste afnemer van het **Legacy** HTML/CSS component Panel hier geen gebruik meer van maakt.

@iterox wil jij `packages/dso-toolkit/src/legacy/legacy-panel.scss` verwijderen en een BREAKING release uitbrengen? Dit component is volgens de oude deprecation policy al eens deprecated. Er hoeft dus geen intermediate deprecate release uit maar dit kan direct worden verwijderd.

Volgens mij is er ook nergens meer documentatie die verwijst naar deze oude selector.

Er is wel een nieuw component "Panel" waar alleen maar een Web Component implementatie van is opgeleverd. Dit is een ander component dan "Legacy Panel" (overigens heette dit component eerst Panel).
